src/cache/cache_las.c

```diff
@@ -779,34 +844,9 @@ __wt_las_insert_updates(WT_CURSOR *cursor, WT_BTREE *btree, WT_PAGE *page, WT_MU
             if (upd->prepare_state == WT_PREPARE_INPROGRESS)
                 ++prepared_insert_cnt;
 
-            /*
-             * If we encounter a modify with preceding updates that have the same timestamp and
-             * transaction id, they need to be squashed into a single lookaside entry to avoid
-             * displacing each other.
-             */
-            if (upd->type == WT_UPDATE_MODIFY && upd->next != NULL &&
-              upd->start_ts == upd->next->start_ts && upd->txnid == upd->next->txnid) {
-                WT_ERR(__las_squash_modifies(session, cursor, &upd));
-                ++insert_cnt;
```

Comment:
Is the insert counter going to be different in the new code layout?

Reply:
I don't see a need to this counter along with other counters such as `las_remove_count` and their stats along with `txn_prepared_updates_lookaside_inserts`. Earlier these counters are used to identify how many las records are present, but now as we are moving into a different way of cleanup, there is no way to identify how many records are present. I prefer to remove them entirely. Comments?

Reply:
Removing them entirely makes sense if we don't need them any more. I'd rather do that than subtly change what they mean.

Reply:
Currently, there is a usage for the `insert_cnt` variable in the las_sweep. Created a new ticket to take care of removing the `insert_cnt` once the las_sweep code is merged.